polls/tests.py

```diff
@@ -33,3 +36,57 @@ def test_check_user_lock_no_redirect(self):
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_302_FOUND)
         self.assertTemplateUsed("polls/task_flow.html")
+
+
+class LockUserAnnotationListTest(APITestCase):
+    """This class sets up the test db and tests
+    - 'lock-users-api' endpoint
+    - 'annotation-api' endpoint
+    """
+
+    def setUp(self):
+        self.user = get_user_model().objects.create(
+            username="test_user", password="testpass"
+        )
```

Comment:
Password is not saved into hash, so user is can't authenticate.

Reply:
Kindly check the last commit, all these has been fixed.